nifi-assembly/pom.xml

```diff
@@ -350,6 +350,42 @@ language governing permissions and limitations under the License. -->
             <version>2.0.0-SNAPSHOT</version>
             <type>nar</type>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-generic-db-adapter-provider-nar</artifactId>
+            <version>2.0.0-SNAPSHOT</version>
+            <type>nar</type>
```

Comment:
Can you share more about the benefit of each of these being their own nar?  If i understand each nar correctly it does not contain provider specific things like mssql/etc..  So it seems like they can all be together.  If something had bespoke requirements splitting them is good.

Can you share more on what you had in mind?

Reply:
I was not sure about what granularity would be accaptable. I'm fine with combining them all together in a single controller service. The user could then select the needed dialect from a dropdown.

Reply:
@exceptionfactory Sounds reasonable. I'll try to apply this concept to my PR soon.